lib/src/main/java/graphql/nadel/engine/transform/result/json/JsonNodes.kt

```diff
@@ -11,6 +13,9 @@ import java.util.concurrent.ConcurrentHashMap
  *
  * Use [NadelCachingJsonNodes] for the most part because that is faster.
  * It is the default implementation.
+ *
+ * @param data The JSON map data.
+ * @param pathPrefix For incremental (defer) payloads, this is the prefix that needs to be removed from the path.
```

Comment:
Wait this comment doesn't belong here…

Should be on `JsonNodes.Companion.invoke` or so